shared/chat/conversation/messages/reactions-row/index.js

```diff
@@ -98,8 +98,6 @@ const styles = styleSheetCreate({
   container: {
     alignItems: 'flex-start',
     flexWrap: 'wrap',
-    // refer to `WrapperAuthor` styles
-    marginLeft: 32 + globalMargins.tiny + (isMobile ? globalMargins.tiny : globalMargins.small),
```

Comment:
things under messages now have a padding applied at a higher level so this kind of hackiness can go away